### ISSUE TITLE
Automatically set render style when adding effect to DMX model group

### ISSUE THREAD

**Issue by schreyack** [xLightsSequencer/xLights#4895]
**Is your feature request related to a problem? Please describe.**
It is annoying to always have to manually set the render style to Per Model Default for Model Groups that contain DMX models especially when copying an effect.

**Describe the solution you'd like**
An option in Settings that when enabled, will automatically set the Layer Settings -> Render Style to Per Model Default any time an effect is added to a Model Group that contains DMX models.  This should happen no matter how the effect is added (add new, copy/paste, move, etc...)

**Describe alternatives you've considered**
Keep doing it by hand.  And then forget.  And then think why is that not rendering right.  And then remember, oh right, gotta go set the Render Style. Grrrrrr.  Haha. :) 

**Additional context**
It's a small thing, but it would be super nice!

**Comment by schreyack** [comment]
Or, even better, is it possible to make DMX models work correctly with all render styles?  It is quite limiting to not be able to put effects on a group of DMX lights.

**Comment by cybercop23** [comment]
Not really, that's why there are diff types and you choose what makes sense for that instance.
But, couldn't you set the default layout mode for the group to Default Model As a Pixel in the group properties?

**Comment by schreyack** [comment]
You can, but it doesn't seem to make any difference in the way rendering occurs. The issue is that with most effects when you apply them to a group of DMX (with per model default set) all the DMX lights will do exactly the same thing.  So, for example, if you apply a Plasma effect, each light will show exactly the same color vs a blend of colors like you would expect with a Plasma effect.  Does that make sense?

It results in having to essentially manually apply effects to each individual DMX light to get the desired result vs being able to just apply the effect once to a group of them.
